pkg/tfvars/openstack/openstack.go

```diff
@@ -79,7 +85,30 @@ func TFVars(masterConfig *v1alpha1.OpenstackProviderSpec, cloud string, external
 			return nil, err
 		}
 
-		cfg.BaseImageLocalFilePath = localFilePath
+		// OpenStack Glance cannot work with compressed data, so we have to check whether or not we need
```

Comment:
Nit: future releases of OpenStack (starting with train) will be able to work with [compressed images](https://specs.openstack.org/openstack/glance-specs/specs/train/approved/glance/add-new-container-format.html). Could you perhaps add a note as "Possible future improvement"? I'm sure a lot of people would appreciate to only have to upload 700MB instead of 2GB if their glance service supports this feature.

Reply:
Yep, I'll update the patch in a minute